### PR TITLE
create present value module

### PR DESCRIPTION
**Pull request by alirezasariri78**
- [x] I have performed a self-review of my code
- [x] My code follows the style guidelines of this project
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Comments in areas I changed are up to date
- [x] I have added comments to hard-to-understand areas of my code
- [ ] I have made corresponding changes to the README.md

### CODE REVIEW

**Review by Esk-C — APPROVED** [review]

File: Algorithms.Tests/Financial/PresentValueTests.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using Algorithms.Financial;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Financial;
+
+public static class PresentValueTests
+{
+    [Test]
+    public static void Present_Value_General_Tests()
```
Comment (siriak) [inline Algorithms.Tests/Financial/PresentValueTests.cs:12]: Please use TestCase to create parameterized tests

**Review by siriak — CHANGES_REQUESTED** [review]

File: Algorithms.Tests/Financial/PresentValueTests.cs (hunk 1 above)
Comment (alirezasariri78) [reply Algorithms.Tests/Financial/PresentValueTests.cs]: done

**Comment by siriak** [comment]
Please also cover this branch 
image

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Comment by alirezasariri78** [comment]
done

**Pull request merged**